src/System.Linq.Expressions/tests/System.Linq.Expressions.Tests.csproj

```diff
@@ -23,6 +23,9 @@
     <Compile Include="$(CommonTestPath)\System\TypeBuilderExtensions.cs" />
   </ItemGroup>
   <ItemGroup>
+      <CodeCoverageAssemblies Include="Microsoft.CSharp" /> 
```

Comment:
Since this PR is still open it's as good a test as any of #15441 working, so I changed this PR to use it, and it looks good.